wdl/filter_variants.wdl

```diff
@@ -404,11 +406,41 @@ task FilterVariantCalls {
             --filter-expression 'GERMQ < ~{min_not_germline_quality}' \
             --filter-name "germline" \
             --filter-expression 'POPAF < ~{germline_min_population_af}' \
+            --filter-name "RESCUED" \
+            --filter-expression "vc.isFiltered() && ((vc.getFilters().size()==1 && vc.getFilters().contains('strand_bias')) || (vc.getFilters().size()==2 && vc.getFilters().contains('strand_bias') && vc.getFilters().contains('weak_evidence'))) && (ROQ >= 30) && (TLOD >= 2.4)" \
             ~{if (length(filter_names) > 0) then " --filter-name '" else ""}~{default="" sep="' --filter-name '" filter_names}~{if (length(filter_names) > 0) then "'" else ""} \
             ~{if (length(filter_expressions) > 0) then " --filter-expression '" else ""}~{default="" sep="' --filter-expression '" filter_expressions}~{if (length(filter_expressions) > 0) then "'" else ""} \
             --output '~{output_vcf}' \
             ~{variant_filtration_extra_args} \
             2> >(grep -v "WARN  JexlEngine - " >&2)
+        
+        OUT_VCF="~{output_vcf}"
+
+        # Need to check if output is compressed or not
+        if [ "~{compress_output}" = "true" ]; then
+            # Decompress to temporary plain VCF
+            TMP_VCF="~{dollar}{OUT_VCF%.gz}"   # strip .gz → *.vcf
+            gunzip -c "~{dollar}{OUT_VCF}" > "~{dollar}{TMP_VCF}"
+
+            # Edit FILTER column on the plain VCF
+            sed -i 's/\tRESCUED;strand_bias\t/\tRESCUED\t/g' "~{dollar}{TMP_VCF}"
+            sed -i 's/\tRESCUED;strand_bias;weak_evidence\t/\tRESCUED\t/g' "~{dollar}{TMP_VCF}"
+
+            # Recompress with bgzip and overwrite original OUT_VCF
+            bgzip -f "~{dollar}{TMP_VCF}"
+            mv "~{dollar}{TMP_VCF}.gz" "~{dollar}{OUT_VCF}"
+            tabix -f -p vcf "~{dollar}{OUT_VCF}"
+        else
+            # Edit FILTER column directly on the plain VCF
+            sed -i 's/\tRESCUED;strand_bias\t/\tRESCUED\t/g' "~{dollar}{OUT_VCF}"
+            sed -i 's/\tRESCUED;strand_bias;weak_evidence\t/\tRESCUED\t/g' "~{dollar}{OUT_VCF}"
+
+            # Compress and index for MSSW
+            bgzip -f "~{dollar}{OUT_VCF}"
```

Comment:
"compress_output" is a flag where the user requests gzipped output or not. Gzipping here will always return gzipped output even if the user requests uncompressed vcf output. @garrett-lam